packages/web-react/src/components/Field/ValidationText.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ElementType } from 'react';
 import { ValidationTextType } from '../../types';
 
-export interface ValidationTextProps {
+interface ValidationTextProps {
```

Comment:
Missing `export`?

Reply:
🤔 this should extend `ValidationTextProps` from `shared` but what about the name?

Reply:
Haha, yes, I decided not to export it, because it was clashing with another name. I wasn't sure how to name it, to keep it consistent and this was an easy solution.

Reply:
Ok, I have fixed this.